CoursesParser/BaseParser.cs

```diff
@@ -0,0 +1,199 @@
+using CoursesParser.Enums;
+using FRITeam.Swapify.Backend.CourseParser;
+using HtmlAgilityPack;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Net;
+using System.Text;
+
+namespace CoursesParser
+{
+    public class BaseParser
+    {
+        private const string Url = "http://vzdelavanie.uniza.sk/vzdelavanie/plany.php";
+        private HtmlDocument _document;
```

Comment:
why did you remove readonly kewords from here? Why is _document declared as field?